libs/langchain/langchain/llms/vertexai.py

```diff
@@ -139,8 +131,9 @@ def _predict(
         self, prompt: str, stop: Optional[List[str]] = None, **kwargs: Any
     ) -> str:
         params = {**self._default_params, **kwargs}
-        res = completion_with_retry(self, prompt, **params)  # type: ignore
-        return self._enforce_stop_words(res.text, stop)
+        if stop:
+            params["stop_sequences"] = stop
```

Comment:
should we also check if self.stop was specified?

Reply:
Done!